py/torch_tensorrt/dynamo/lowering/_decomposition_groups.py

```diff
@@ -97,10 +97,6 @@
     aten.nan_to_num,
     aten.narrow,
     # TODO: Disable the below operators once freezing is done
-    aten.native_batch_norm_backward,
-    aten._native_batch_norm_legit,
-    aten._native_batch_norm_legit_functional,
-    aten._native_batch_norm_legit_no_training,
```

Comment:
This PR only supports for `aten._native_batch_norm_legit_no_training`. Should we remove others?

Reply:
I think we should only remove `_native_batch_norm_legit_no_training` here, since the other operators could still show up in ATen graphs.

Reply:
done